### PR TITLE
babeld: add page

### PR DESCRIPTION
**Pull request by marek22k**
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.12.1

### CODE REVIEW

File: pages/common/babeld.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# babeld
+
+> Routing daemon for Babel which uses firewall-style filters.
+> More information: <https://www.irif.fr/~jch/software/babel/babeld.html>.
+
+- Start babeld with a specific configuration file:
+
+`babeld -c {{babeld.conf}}`
```
Comment (CleanMachine1) [inline pages/common/babeld.md:8]:
```suggestion
`babeld -c {{path/to/babeld.conf}}`
```

Could this be applied throughout with all files.

File: pages/common/babeld.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start babeld with multiple configuration file (read in order):
+
+`babeld -c {{ports.conf}} -c {{filters.conf}} -c {{interfaces.conf}}`
+
+- Start babeld and demonize afterwards:
```
Comment (CleanMachine1) [inline pages/common/babeld.md:14]:
```suggestion
- Start babeld and daemonise afterwards:
```

File: pages/common/babeld.md (hunk 1 above)
Comment (marek22k) [reply pages/common/babeld.md]: Line 12 is then a bit long. Is that okay?

File: pages/common/babeld.md (hunk 2 above)
Comment (marek22k) [reply pages/common/babeld.md]: Thanks!

File: pages/common/babeld.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start babeld and daemonise afterwards:
+
+`babeld -D`
+
+- Start babeld and pass a configuration command:
+
+`babeld -C {{'redistribute metric 256'}}`
+
+- Start babeld and specify on which interfaces to operate:
```
Comment (CleanMachine1) [inline pages/common/babeld.md:22]:
```suggestion
- Start babeld and specify which network interfaces to operate:
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/babeld.md (hunk 3 above)
Comment (marek22k) [reply pages/common/babeld.md:22]: Maybe `Start babeld and specify which network interfaces to operate on:`. I think with the example and the fact that it is a routing daemon, it is understandable that babeld are dealing with network interfaces. I made the sentence so short that it looks nicer.

File: pages/common/babeld.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/babeld.md]: Fine.

File: pages/common/babeld.md (hunk 3 above)
Comment (CleanMachine1) [reply pages/common/babeld.md:22]: Yes, you are right, edited to include the on.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

File: pages/common/babeld.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# babeld
+
+> Routing daemon for Babel which uses firewall-style filters.
+> More information: <https://www.irif.fr/~jch/software/babel/babeld.html>.
+
+- Start babeld with a specific configuration file:
+
+`babeld -c {{path/to/babeld.conf}}`
+
+- Start babeld with multiple configuration file (read in order):
```
Comment (navarroaxel) [inline pages/common/babeld.md:10]: more than 1, then
```suggestion
- Start babeld with multiple configuration files (read in order):
```

Comment (marek22k) [reply pages/common/babeld.md]: Thanks for the hint!

**Pull request merged**